packages/terra-form-select/src/_Menu.jsx

```diff
@@ -490,17 +490,17 @@ class Menu extends React.Component {
    * Options below the dropdown will be scrolled to the bottom of the menu.
    */
   scrollIntoView() {
-    if (!this.state.active) {
+    const activeOption = this.menu.querySelector('[data-select-active]');
+    if (!this.state.active || activeOption === null) {
       return;
     }
```

Comment:
Can you provide steps to reproduce the issue this PR solves?

Something might be out of sync, if the state claims there is an active element, it should be possible to query for it. Something suspicious might be going on.

Reply:
I attempted to outline it above. The short version for us was:
- **Empty** select that calls some service to that returns data that becomes options
- Click into select, before service returns, type in anything
- When service returns, it attempts to render new options, and fails